yamale/tests/fixtures/map.yaml

```diff
@@ -1 +1,3 @@
 map: map(str(), int())
+min: map(int(), min = 1)
+max: map(int(), max = 2)
```

Comment:
Can you add a test which includes both min and max?

Reply:
Done!